gg-pingpong-api/src/main/resources/db/migration/V1.3__update_party_content.sql

```diff
@@ -0,0 +1,105 @@
+CREATE TABLE category (
+    id BIGINT PRIMARY KEY AUTO_INCREMENT,
+    category_name VARCHAR(10) NOT NULL,
+    created_at DATETIME,
+    modified_at DATETIME
+);
+
+CREATE TABLE room (
+    id BIGINT PRIMARY KEY AUTO_INCREMENT,
+    host_id BIGINT,
+    creator_id BIGINT,
+    category_id BIGINT,
+    title VARCHAR(15),
+    content VARCHAR(100),
+    max_people INT,
+    min_people INT,
+    due_date DATETIME,
+    created_at DATETIME,
+    modified_at DATETIME,
+    room_status VARCHAR(10),
+    FOREIGN KEY (host_id) REFERENCES User(id),
+    FOREIGN KEY (creator_id) REFERENCES User(id),
+    FOREIGN KEY (category_id) REFERENCES Category(id)
+);
+
+CREATE TABLE userroom (
+    id BIGINT PRIMARY KEY AUTO_INCREMENT,
+    user_id BIGINT,
+    room_id BIGINT,
+    nickname VARCHAR(20),
+    is_exist BOOLEAN,
+    created_at DATETIME,
+    modified_at DATETIME,
+    FOREIGN KEY (user_id) REFERENCES user(id),
+    FOREIGN KEY (room_id) REFERENCES room(id)
+);
+
+CREATE TABLE comment (
+    id BIGINT PRIMARY KEY AUTO_INCREMENT,
+    user_id BIGINT,
+    userroom_id BIGINT,
+    room_id BIGINT,
+    content VARCHAR(100),
+    is_hidden BOOLEAN,
+    created_at DATETIME,
+    modified_at DATETIME,
+    FOREIGN KEY (user_id) REFERENCES User(id),
+    FOREIGN KEY (userroom_id) REFERENCES UserRoom(id),
```

Comment:
```suggestion
    FOREIGN KEY (user_room_id) REFERENCES UserRoom(id),
```
snake case 사용하면 더 좋을 것 같아요